integration_tests/commands/type_test.go

```diff
@@ -0,0 +1,57 @@
+package commands
+
+import (
+	"testing"
+
+	dstore "github.com/dicedb/dice/internal/store"
+	"gotest.tools/v3/assert"
+)
+
+func TestType(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
+	testCases := []struct {
+		name     string
+		commands []string
+		expected []interface{}
+		setup    func(store *dstore.Store)
```

Comment:
Just a question, why are we using store directly here in integration tests. Any setup we want to do can be done using FireCommand(conn,"   " )

Reply:
There was a misunderstanding from my side. We don't actually need to store directly in integration tests. Initially, I was considering storing the zset and stream objects directly, without using commands that aren't natively supported in DiceDB. However, I later realized that we only need to support string, list, set, and hash types in the Type command.